backend/cmmgen.ml

```diff
@@ -408,15 +408,22 @@ let rec transl env e =
         | f :: rem ->
             Cmmgen_state.add_function f;
             let dbg = f.dbg in
+            let arity =
+              if f.arity >= 0 then
```

Comment:
Instead, I'll delete it in the next PR. (Once the middle-end logic is in place, f.arity will actually be a Clambda.arity, rather than doing this weird negative-encoding trick)